blog/2022-03-21-vxt-to-join-marketplace.md

```diff
@@ -0,0 +1,40 @@
+---
+slug: vxt-joins-marketplace
+title: "Vxt is on a mission to eradicate boring tasks. Now they’re joining Working Group Two’s product marketplace."
+date: 2022-03-21
+tags: [marketplace, voicemail, partners]
+authors: [marius-waldum]
+---
+
+**Vxt**, a technology communications company in New Zealand, will be adding their voicemail product to Working Group Two’s marketplace. Their aim is to increase visibility and distribution with mobile operators on Working Group Two’s platform. 
+
+<!--truncate-->
+
+The passionate [Vxt](www.vxt.co.nz) Vxt team is on a mission to leverage communications data to automate the most boring work in the world, namely administrative tasks. They’ve made voicemail both simpler and smarter by offering instant transcriptions, smart greetings, email integration, and the ability to access your messages from anywhere. 
+
+But wait, isn’t voicemail a thing of the past? 
+
+_We see the importance of voicemail increasing or declining based on specific demographics,”_ says Luke Campbell, the CEO of Vxt. 
+
+Vxt describes people who are highly dependent on their phones for their work as being particularly passionate about voicemail. Lawyers, real estate agents, plumbers - they might get dozens of calls every day. So where others turn off voicemail, these people might want to get their voicemails forwarded to their emails, be able to share their inbox with a colleague or assistant, have voicemails automatically translated to text, and integrate their messages with productivity tools like Asana or an ERP system. Some may also need to have their messages stored for regulatory or security purposes. A real estate agent with (say) 10,000 contacts, may want to customize the greetings so that every time one of the 200 Johns or Jessicas call, they get a personal message stating their name. 
+
+Soon, Vxt Voicemail will be added to Working Group Two’s marketplace and made available to operators on the platform. The operators will then be able to offer Vxt to their customers. 
+
+_“It’s exciting to work with companies like Vxt. They have set out to solve real pain points by automating routine tasks such as calling, voicemail, and note-taking. We have a shared view on the importance of building scalable ecosystems and programmable networks that leverage built-in core functionality in a new way,”_ says Marius Waldum, the Head of Product at Working Group Two. 
+
+The Working Group Two marketplace is an integrated part of the company’s ambition to deliver value for operators, product developers, and mobile customers alike. Working Group Two’s cloud-native mobile network brings radical simplification, cost efficiency and a platform to leverage a developer community - to drive innovation and capitalize on disruptive technologies.
+
+To learn more, visit https://www.wgtwo.com/product-ecosystem/ or get in touch with us at products@wgtwo.com.
```

Comment:
Perhaps with some neat links?

`To learn more, visit [our products ecosystem page](https://www.wgtwo.com/product-ecosystem/) or get in touch with us at [products@wgtwo.com](mailto:products@wgtwo.com).`

Reply:
Excellent. Can I just directly accept this edit somehow in gh? (sorry, I'm still learning)